orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/BuildService.java

```diff
@@ -56,12 +56,4 @@ public Map queuedBuild(String master, String item) {
       Integer buildNumber, String fileName, String master, String job) {
     return igorService.getArtifacts(buildNumber, fileName, master, encode(job));
   }
-
-  public List compareCommits(
```

Comment:
I moved this to the new `ScmController` because, well, it doesn't have anything to do with builds. 😃